### PR TITLE
Feature/create file from textbox

### PR DESCRIPTION
**Pull request by klaseskilson**
This PR features the following:
- Ruby version 2.2.1 and Rails version 4.2.1
- A textarea that can create links and text files (only available if the logged in user has a local identity)

### CODE REVIEW

**Comment by ronjagrosz** [comment]
:+1:

File: app/assets/javascripts/controllers/CreateFromTextCtrl.coffee [hunk 1]
```diff
@@ -0,0 +1,31 @@
+@bruseApp.controller 'CreateFromTextCtrl', ['$scope', '$http', ($scope, $http) ->
+  $scope.text = ''
+  $scope.loading = false
+
+  $scope.$watch "text", (newText, oldText) ->
+    return unless angular.isDefined(newText) && angular.isDefined(oldText)
+    # regex is afwul but useful, look here to understand what it does: http://www.regexr.com/3aqpu
+    urlPattern = new RegExp('(https?|s?ftp):\/\/[\w-]*(\.[\w-]*)+([\w.,@?^=%&amp;:/~+#-]*[\w@?^=%&amp;/~+#-])?')
+    # this regex matches any ws char
```
Comment (tkomstadius) [inline app/assets/javascripts/controllers/CreateFromTextCtrl.coffee:9]: what does ws stand for?

Comment (klaseskilson) [inline app/assets/javascripts/controllers/CreateFromTextCtrl.coffee:9]: White space! Newline charactares and spaces basically.

File: app/controllers/files/files_controller.rb [hunk 2]
```diff
@@ -85,4 +103,73 @@ def set_file
...
+    end
+
+    # Private: Create a text file containing content
+    #
+    # content   - the file content
+    #
+    # Returns a new BruseFile
+    def create_text_file(content)
+      # generate file name
+      fileref = SecureRandom.uuid
```
Comment (tkomstadius) [inline app/controllers/files/files_controller.rb:135]: shouldn't that also have 10 chars or is that changed?

Comment (klaseskilson) [inline app/controllers/files/files_controller.rb:135]: [`SecureRandom.uuid` doesn't accept any parameters!](http://ruby-doc.org/stdlib-2.2.2/libdoc/securerandom/rdoc/SecureRandom.html#method-c-uuid) It's 100% automagic.

**Comment by tkomstadius** [comment]
I can't find the textbox

File: app/controllers/files/files_controller.rb (hunk 2 above)
Comment (tkomstadius) [inline app/controllers/files/files_controller.rb:135]: oh, it was the hex I was thinking about

**Comment by klaseskilson** [comment]
It should be available from the home template. So if you go to http://localhost:3000/ you should see it there.

**Comment by tkomstadius** [comment]
I don't

**Comment by tkomstadius** [comment]
found it with my local lidentity. hehe.. looks najs. merge

**Pull request merged**